build/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/GlidingTracks/gt-backend/constant"
 	"github.com/GlidingTracks/gt-backend/rest"
 	"github.com/gorilla/mux"
+	"github.com/rs/cors"
```

Comment:
You have to add the library to the vendor or else heroku doesn't know about it. https://github.com/kardianos/govendor